x-pack/metricbeat/module/enterprisesearch/stats/_meta/fields.yml

```diff
@@ -231,6 +231,157 @@
           type: long
           description: Total number of jobs waiting in the failed queue.
 
+
+    - name: crawler
+      type: group
+      description: Aggregate stats on the functioning of the crawler subsystem within Enterprise Search.
+      fields:
+        - name: global
+          type: group
+          description: Global deployment-wide metrics for the crawler.
+          fields:
+            - name: crawl_requests
+              type: group
+              description: Crawl request summary for the deployment.
+              fields:
+                - name: pending
+                  type: long
+                  description: Total number of crawl requests waiting to be processed.
+                - name: active
+                  type: long
+                  description: Total number of crawl requests cyrrently being processed (running crawls).
+                - name: successful
+                  type: long
+                  description: Total number of crawl requests that have succeeded.
+                - name: failed
+                  type: long
+                  description: Total number of failed crawl requests.
+
+        - name: node
+          type: group
+          description: Node-level statistics for the crawler.
+          fields:
+            - name: pages_visited
+              type: long
+              description: Total number of pages visited by the crawler since the instance start.
+
+            - name: urls_allowed
+              type: long
+              description: Total number of URLs allowed by the crawler during discovery since the instance start.
+
+            - name: urls_denied
+              type: group
+              description: Total number of URLs debied by the crawler during discovery since the instance start, broken down by deny reason.
```

Comment:
```suggestion
              description: Total number of URLs denied by the crawler during discovery since the instance start, broken down by deny reason.
```